bindings/wasm/examples/src/web.js

```diff
@@ -83,4 +63,4 @@ window.onload = async () => {
         .querySelector("#did_history_btn")
         .addEventListener("click", () => resolveHistory(CLIENT_CONFIG));
 
-};
+};
```

Comment:
Nit: add back the trailing newline at the end of files, configure the IDE to do it automatically.